login-workflow/src/new-architecture/screens/AccountDetailsScreen/types.ts

```diff
@@ -0,0 +1,16 @@
+import { TextFieldProps } from '@mui/material/TextField';
+import { WorkflowCardProps } from '../../components/WorkflowCard/WorkflowCard.types';
+
+export type AccountDetailsScreenProps = WorkflowCardProps & {
+    // used to test the input for valid formatting
+    firstNameLabel?: string;
+    initialFirstName?: string;
+    firstNameValidator?: (firstName: string) => boolean | string;
+    firstNameTextFieldProps?: Omit<TextFieldProps, 'label' | 'inputRef' | 'value' | 'onChange' | 'onKeyPress'>;
+
```

Comment:
need to verify is we need to expose the isFirstNameRequired? and isLastNameRequired? to turn off the validator if the user does not want the validator

Reply:
note from story - When the user input in a field changes, the respective validator function should be called (if provided)

Reply:
We have removed the `isFirstNameRequired` and `isLastNameRequired` props as it will get validated in the respective callback functions. @joebochill

Reply:
Correct, the required/not-required nature of inputs will be controlled through the TextFieldProps instead